cmd/do-agent/config.go

```diff
@@ -164,6 +164,20 @@ func initDecorator() decorate.Chain {
 	return chain
 }
 
+// initAggregatorSpecs initializes the field aggregation specifications.
+// The map's key is the prometheus metric name to aggregate over, and the value is the label to aggregate away.
+func initAggregatorSpecs() map[string]string {
+	aggregateSpecs := make(map[string]string)
+
+	if config.dbaas != "" {
+		for k, v := range dbaasAggregationSpec {
+			aggregateSpecs[k] = v
+		}
+	}
+
+	return aggregateSpecs
```

Comment:
It seems this function just copies the map to another location. Is the idea to create a union of aggregators when more will be used in the future? Meaning, this only here to allow adding more `if config.<something else> != "" { union }`?

Reply:
yes, so we can add in other aggregate specs when another flag is enabled